### DISCUSSION TITLE
permissions

### DISCUSSION THREAD

**Discussion by rusheedroberts** [snipe/snipe-it#10489] [Q&A]
Why can’t my user who has been given permissions to [View, Create, Edit, Delete, and Checkout] – Consumables see any of the current existing Consumables in the list? Instead, all they see is this. Even if they add a new item, while it saves successfully, it is not visible to the user. 

The full list of Consumables only becomes visible if the user is promoted to the classification of “ Super User”

![screen shot](https://user-images.githubusercontent.com/77894940/148591977-43a69cac-6202-43df-8af0-802cbd8cad47.png)

**Comment by yosiasz** [comment]
> Why can’t my user who has been given permissions to [View, Create, Edit, Delete, and Checkout] – Consumables see any of the current existing Consumables in the list? Instead, all they see is this. Even if they add a new item, while it saves successfully, it is not visible to the user.
> 
> The full list of Consumables only becomes visible if the user is promoted to the classification of “ Super User”
> 
> ![screen shot](https://user-images.githubusercontent.com/77894940/148591977-43a69cac-6202-43df-8af0-802cbd8cad47.png)

have you implemented multi company settings? and which company is this user part of?
